authors/apps/comments/renderers.py

```diff
@@ -0,0 +1,22 @@
+import json
+from rest_framework.renderers import JSONRenderer
+from rest_framework.utils.serializer_helpers import ReturnDict, ReturnList
```

Comment:
Thanks for the great work @wasibani-roy, However you imported ReturnDict that you did not use. you can always refactor the code to improve code readability.